script/proto-docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+mkdir -p docs/grpc
+
+DOCROOT="${DOCROOT:$PWD}"
+target=docs/grpc
+templates=docs/grpc-templates
+filemap="$templates/filemap.xml"
+
+cp "$templates/index-header.html" "$target/index.html"
+
+
+for pkg in \
```

Comment:
Could something like `find api -name '*proto*' -printf "%h\n" | sort | uniq` be used so we don't have to update the script when new services are added?

Reply:
Ya, that looks good